deploy/services-demo/conf/nginz/zauth_acl.txt

```diff
@@ -4,10 +4,13 @@ a (blacklist (path "/provider")
              (path "/bot/**")
              (path "/i/**"))
 
-u (whitelist (path "/access"))
```

Comment:
why?

Reply:
(hm, ok, somehow this must have to do with the fact that we're using nginz in integration tests now.  perhaps it's easier if you explain this to me in person / on the phone.)

(i would also like to talk about the rust code, since i'm not very familiar with that code base.  i'll see if you have time tomorrow.)

Reply:
This is actually unrelated to integration tests.

whitelisting /access for user tokens has no effect, as that route is not zauth-authenticated. This line has been in the code for a few years without effect, so we might as well remove it. I noticed this having no effect due to a new integration test that also talks to nginz.